vagrant/docker_slave/Vagrantfile

```diff
@@ -0,0 +1,17 @@
+require '../shared.rb'
```

Comment:
Nah, this is what code looks like when it's written The Correct Way.

With tabs, instead of spaces. Now excuse me while I go fan some other flames over here...

(PR updated with spaces instead of tabs)